es.hjson

```diff
@@ -591,66 +591,66 @@
+    "allWarnsRemovedAnswerText": "Advertencias anteriores eliminadas"
+    "chooseMaxWarnsValue": "<b>¿A partir de qué número de advertencias se debe banear a un usuario?</b>"
     "chooseMaxWarnsValueAnswerText": "Seleccione un valor"
-    "maxWarnsValueChangedBy": "<b>Max Warn cambió de </b><code>%s</code><b> a </b><code>%s</code><b> por </b>%s"
-    "maxWarnsValueChangedAnswerText": "Nuevo conjunto Max Warn"
+    "maxWarnsValueChangedBy": "<b>Advertencias máximas establecidas de </b><code>%s</code><b> a </b><code>%s</code><b> por </b>%s"
+    "maxWarnsValueChangedAnswerText": "Nuevas advertencias máximas establecidas"
   }
 
-  "warn.noReasonGiven": "Ninguna razón dada"
+  "warn.noReasonGiven": "Ningún motivo dado"
 
-  "warn.replyHint": "<b>Responda al usuario al que desea advertir, por ejemplo:</b>\n\n<code>/warn</code><b> o\n</b><code>/warn &lt;razón aquí&gt;</code>"
+  "warn.replyHint": "<b>Responda al usuario al que desea advertir, por ejemplo:</b>\n\n<code>/warn</code><b> o\n</b><code>/warn &lt;motivo aquí&gt;</code>"
 
-  "warn.success": "%s<b> fue advertido por </b>%s.%s\n\n<code>%s fuera de %s advierte</code>"
-  "warn.userBanned": "%s<b> fue prohibido. Alcanzado </b><code>%s fuera de %s advierte</code>"
-  "warn.myWarns": "<b>Advierte sobre </b>%s<b>:</b>"
-  "warn.noWarns": "No advierte, muy inteligente."
+  "warn.success": "%s<b> fue advertido por </b>%s.%s\n\n<code>%s de %s advertencias</code>"
+  "warn.userBanned": "%s<b> fue baneado. Alcanzó </b><code>%s de %s advertencias</code>"
+  "warn.myWarns": "<b>Advertencias para </b>%s<b>:</b>"
+  "warn.noWarns": "No hay advertencias, muy inteligente."
 
 
   /* ---------------------------------------------------------------------
                             Start - Used for /start
      --------------------------------------------------------------------- */
   # ⚠️ IMPORTANT
-  "start.error.JoinVerifyWrongUser": "<b>Ese botón no debe ser presionado por ti.\n\nPero bueno, ahora que estás aquí:</b>"
-  "start.error.JoinVerifyExpired": "<b>Lo sentimos, su solicitud de verificación de unión ha caducado.\n\nVuelva a unirse al grupo e intente nuevamente.</b>"
+  "start.error.JoinVerifyWrongUser": "<b>Ese botón no debe ser presionado por ti.\n\nPero bueno, ahora ya estás aquí:</b>"
+  "start.error.JoinVerifyExpired": "<b>Lo sentimos, su solicitud de verificación de unirse al grupo ha caducado.\n\nVuelva a ingresar al grupo e intente nuevamente.</b>"
   "start.main":
   '''
-  <b>Hola </b><code>%s</code><b>, soy %s!
+  <b>¡Hola </b><code>%s</code><b>, soy %s!
 
-  ✴️ Mantengo a tu grupo a salvo del mal</b> y también tengo <b>muchas otras características interesantes</b> como <a href="t.me/CalsiBotDev/48">traducciones</a><b>.</b>
+  ✴️ I keep your group safe from evil</b> and also have <b>tons of other cool features</b> like <a href="t.me/CalsiBotDev/48">translations</a><b>.</b>

```

Comment:
This can remain in Spanish, since it is not a translation comment, but actual text that will be shown to users.
```suggestion
  ✴️ Mantengo a tu grupo a salvo del mal</b>, y también tengo <b>muchas otras herramientas interesantes</b> como las <a href="t.me/CalsiBotDev/48">traducciones</a><b>.</b>
```